examples/example_endpoints.py

```diff
@@ -142,3 +153,84 @@ def predict(
 
     print(answers)
     # TODO what type answers are?
+
+
+@persistent_cache('evaluate')
+def evaluate(
+    dataset_id: Optional[UUID] = None,
+    dataset_name: Optional[str] = None,
+    retriever_type: str = 'dpr',
+    dpr_question_encoder: str = 'facebook/dpr-question_encoder-single-nq-base',
+    dpr_context_encoder: str = 'facebook/dpr-ctx_encoder-single-nq-base',
+    reader_encoder: str = 'deepset/roberta-base-squad2',
```

Comment:
Jeszcze dodać `use_gpu`

Reply:
W sumie to warto zacząć to pisać wśród rzeczywistych endpoint'ów, bo tu już trochę przestarzałe :)